test/testenv/appframework_utils.go

```diff
@@ -130,3 +131,16 @@ func GetAppFileList(appList []string, version int) []string {
 	}
 	return appFileList
 }
+
+// GetAppframeworkManualUpdateConfigMap gets config map for given manual update configmap
```

Comment:
As the config map name is static this method is redundant. Why not directly call GetConfigMap. 
I have used the wrapper in Monitoring Console branch because of config map name dependency on CR name.